src/extension.ts

```diff
@@ -69,9 +71,48 @@ function npmCheck() {
   });
 }
 
+function getPackageManagerConfig(): PackageManagerConfiguration {
+  const config = vscode.workspace.getConfiguration().inspect('npm.packageManager');
+
+  switch (config?.globalValue) {
+    case 'yarn':
+      return 'yarn';
+    case 'npm':
+      return 'npm';
+    default:
+      return undefined;
+  }
+}
+
+function setPackageManagerConfig(value: PackageManagerConfiguration) {
+  vscode.workspace.getConfiguration().update('npm.packageManager', value, true);
+}
+
+function executeInstallTask() {
+  const packageManagerConfig = getPackageManagerConfig();
+  const repoType = REPO_TYPE.YARN ? 'yarn' : 'npm';
+
+  if (packageManagerConfig === undefined) {
```

Comment:
The choice is : 
- we override global config whatever if it already exists or not (and the install dependencies will use always the good type of install)
- we add global config only if it does not exist (but the install dependencies won't use the good type of install in certains cases)

Reply:
Got it. 👍 
Can you please change the commit message to `feat: add install button in prompt`.